nanome/_internal/_structure/_atom.py

```diff
@@ -113,6 +115,15 @@ def _position(self):
     def _position(self, value):
         self._positions[self._current_conformer] = value
 
+    @property
+    def _vdw_radius(self):
+        if len(_Atom._vdw_radii) < 1:
+            _Atom._fill_atom_table()
+        if self._symbol in _Atom._vdw_radii:
```

Comment:
I think we should normalize all the symbols to lower case, and use to lower here, as a safety precaution. Nanome does it